saas/models.py

```diff
@@ -1528,6 +1528,26 @@ def provider(invoiced_items):
             result = get_current_provider()
         return result
 
+    def by_organization(self, organization, transaction_type):
+        return self.filter(
+            # All transactions involving Funds
+            ((Q(orig_organization=organization)
+              & Q(orig_account=transaction_type))
+            | (Q(dest_organization=organization)
+              & Q(dest_account=transaction_type))))
+
+
+    def by_customer(self, organization):
```

Comment:
purely for the record (we talked about it): tabled for now. We will revisit renaming options once we improve on the documentation.